packages/errors/index.js

```diff
@@ -1,4 +0,0 @@
-const css = require('./css')
```

Comment:
Ummm, can we get rid of this file?  Seems like it's still referenced in `errors/package.json`.

Reply:
looks like it is deleted and no longer referenced. I'll check to be sure.